e2e/test/clusterautoscaler/.env

```diff
@@ -0,0 +1,4 @@
+TEST_DIR="e2e/test/clusterautoscaler"
+AZURE_AZS=1
+KEEP_RESOURCES=true
```

Comment:
I guess you don't need to keep the resources after you made sure the code it's fine.

Reply:
Good catch, thanks